deployments/goerli/Mock0x.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0x3721E67EF44Ff2Ab04a96E187382614FeEf6b4Ba",
```

Comment:
This is going to be an issue. The backend only supports one SwapTarget. We don't really want to re-deploy this every time

Reply:
imo the simplest way to deal with the new swaptarget is that you add the new swaptarget to the new contract (because we support the strategy having multiple ones) then we update the swaptarget value in the backend